src/components/Menu/config.ts

```diff
@@ -172,18 +176,14 @@ const config: MenuEntry[] = [
     ],
   },
   // {
-  //   label: 'IFO',
-  //   icon: 'IfoIcon',
-  //   href: 'https://pancakeswap.finance/ifo',
+  //   label: 'IPO',
+  //   icon: 'IpoIcon',
```

Comment:
This relates to [this UIKit PR](https://github.com/pantherswap/panther-uikit/pull/2)